components/Chips/src/MDCChipCollectionViewCell.m

```diff
@@ -100,21 +100,43 @@ - (UIView *)hitTest:(CGPoint)point withEvent:(UIEvent *)event {
 }
 
 - (void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event {
+  if (self.chipView.enableRippleBehavior) {
```

Comment:
Is it important that this method happen before the `super` call? If yes, please add a comment to that effect so maintainers don't move it about thinking the order is not important.

Reply:
Yes it is and is mentioned in the called API header (MDCStatefulRippleView). I've added additional documentation if that wasn't clear.